scanner/findingkey/rootkit.go

```diff
@@ -39,9 +39,14 @@ func (k RootkitKey) RootkitString() string {
 }
 
 func GenerateRootkitKey(info apitypes.RootkitFindingInfo) RootkitKey {
+	message := ""
+	if info.Message != nil {
+		message = *info.Message
+	}
+
 	return RootkitKey{
 		Name:        *info.RootkitName,
 		RootkitType: string(*info.RootkitType),
-		Message:     *info.Message,
+		Message:     message,
```

Comment:
you can use `github.com/openclarity/vmclarity/core/to` module for this via `to.ValueOrZero(info.Message)`

Reply:
Nice!